modules/pricer/src/test/java/com/opengamma/strata/pricer/calibration/CalibrationZeroRateUsd2OisFuturesIrsTest.java

```diff
@@ -0,0 +1,393 @@
+import com.opengamma.strata.market.curve.definition.FixedOvernightSwapCurveNode;
+import com.opengamma.strata.market.curve.definition.IborFixingDepositCurveNode;
+import com.opengamma.strata.market.curve.definition.IborFutureCurveNode;
+import com.opengamma.strata.market.curve.definition.InterpolatedNodalCurveDefinition;
+import com.opengamma.strata.market.curve.definition.TermDepositCurveNode;
+import com.opengamma.strata.market.key.QuoteKey;
+import com.opengamma.strata.market.sensitivity.CurveCurrencyParameterSensitivities;
+import com.opengamma.strata.market.sensitivity.PointSensitivityBuilder;
+import com.opengamma.strata.market.value.ValueType;
+import com.opengamma.strata.math.impl.interpolation.FlatExtrapolator1D;
+import com.opengamma.strata.math.impl.interpolation.LinearInterpolator1D;
+import com.opengamma.strata.pricer.rate.ImmutableRatesProvider;
+import com.opengamma.strata.pricer.rate.deposit.DiscountingIborFixingDepositProductPricer;
+import com.opengamma.strata.pricer.rate.deposit.DiscountingTermDepositProductPricer;
+import com.opengamma.strata.pricer.rate.future.DiscountingIborFutureTradePricer;
+import com.opengamma.strata.pricer.rate.swap.DiscountingSwapProductPricer;
+import com.opengamma.strata.pricer.sensitivity.MarketQuoteSensitivityCalculator;
+
+/**
+ * Test for curve calibration with 2 curves in USD.
+ * One curve is Discounting and Fed Fund forward and the other one is Libor 3M forward.
+ */
+@Test
+public class CalibrationZeroRateUsd2OisFuturesIrsTest {
+
+  private static final LocalDate VALUATION_DATE = LocalDate.of(2015, 7, 21);
+
+  private static final CurveInterpolator INTERPOLATOR_LINEAR = new LinearInterpolator1D();
+  private static final CurveExtrapolator EXTRAPOLATOR_FLAT = new FlatExtrapolator1D();
+  private static final DayCount CURVE_DC = ACT_365F;
+  private static final LocalDateDoubleTimeSeries TS_EMTPY = LocalDateDoubleTimeSeries.empty();
+
+  private static final String SCHEME = "CALIBRATION";
+
+  /** Curve names */
+  private static final String DSCON_NAME = "USD-DSCON-OIS";
+  private static final CurveName DSCON_CURVE_NAME = CurveName.of(DSCON_NAME);
+  private static final String FWD3_NAME = "USD-LIBOR3M-FRAIRS";
+  private static final CurveName FWD3_CURVE_NAME = CurveName.of(FWD3_NAME);
+  /** Curves associations to currencies and indices. */
+  private static final Map<CurveName, Currency> DSC_NAMES = new HashMap<>();
+  private static final Map<CurveName, Set<Index>> IDX_NAMES = new HashMap<>();
+  private static final Map<Index, LocalDateDoubleTimeSeries> TS = new HashMap<>();
+  static {
+    DSC_NAMES.put(DSCON_CURVE_NAME, USD);
+    Set<Index> usdFedFundSet = new HashSet<>();
+    usdFedFundSet.add(USD_FED_FUND);
+    IDX_NAMES.put(DSCON_CURVE_NAME, usdFedFundSet);
+    Set<Index> usdLibor3Set = new HashSet<>();
+    usdLibor3Set.add(USD_LIBOR_3M);
+    IDX_NAMES.put(FWD3_CURVE_NAME, usdLibor3Set);
+    TS.put(USD_FED_FUND, TS_EMTPY);
+    TS.put(USD_LIBOR_3M, TS_EMTPY);
```

Comment:
Time-series are now optional for rate providers, so these are probably not needed.